core/src/main.cpp

```diff
@@ -363,11 +367,11 @@ constexpr auto default_jsonschema() -> std::string_view
                 "resource": {{
                     "type": "string"
                 }},
-                "max_number_of_bytes_per_read_operation": {{
+                "put_object_buffer_size_in_bytes": {{
                     "type": "integer",
                     "minimum": 1
                 }},
-                "buffer_size_in_bytes_for_write_operations": {{
+                "get_object_buffer_size_in_bytes": {{
```

Comment:
Should these be part of the `required` array?

Reply:
I don't think so.  There are defaults if they don't exist.

Reply:
I went ahead and put them in the required array.

Reply:
It's fine to not require them. I was only asking because it wasn't clear to me if they had defaults.

Feel free to remove them if that's more appropriate.